### PR TITLE
ipatests: ipa-healthcheck test for DS BackendsCheck

### PR DESCRIPTION
**Pull request by menonsudhir**
This testcase checks that the BackendsCheck reports the CRITICAL status when dse.ldif present in the
DS instance directory is renamed/moved.

### CODE REVIEW

File: ipatests/test_integration/test_ipahealthcheck.py [hunk 1]
```diff
@@ -967,6 +967,45 @@ def execute_expiring_check(check):
...
+    def rename_ldif(self):
+        """Fixture to rename dse.ldif file and revert after test"""
+        instance = realm_to_serverid(self.master.domain.realm)
+        self.master.run_command(
+            [
+                "mv",
+                paths.ETC_DIRSRV_SLAPD_INSTANCE_TEMPLATE % instance
+                + "/dse.ldif",
+                paths.ETC_DIRSRV_SLAPD_INSTANCE_TEMPLATE % instance
+                + "/_dse.ldif",
```
Comment (rcritten) [inline ipatests/test_integration/test_ipahealthcheck.py:980]: The _dse seems a bit subtle. Can you name it something like dse.ldif.rename or something? I had to stare at this a bit to see the single-character difference.

**Review by rcritten — CHANGES_REQUESTED** [review]
Otherwise it looks good to me.

File: ipatests/test_integration/test_ipahealthcheck.py (hunk 1 above)
Comment (menonsudhir) [reply ipatests/test_integration/test_ipahealthcheck.py]: @rcritten, the file has been changed to dse.ldif.renamed.

**Review by rcritten — APPROVED** [review]

**Comment by rcritten** [comment]
master:

* 9a41966a250a97e7df204b4e66a4e7e9546d78aa ipatests: ipa-healthcheck test for DS BackendsCheck

**Pull request closed without merging**